tools/mysql/README.dev

```diff
@@ -3,5 +3,3 @@ Follow the steps below to update deploy DB schema ( assume current DB schema ver
 1. Create file schema-update-11.sql, and put the DB schema change statements there
 2. Make sure the last statement reads:
    UPDATE schema_versions SET version=11;
-3. Make sure you update the last statement in ../../deploy-service/common/src/main/resources/sql/deploy.sql as well:
```

Comment:
I looked at this file, it seems that we are not using it. We should remove the line from the file as well or should also update it there.

Reply:
looks like schema version is no longer updated per README.dev, i can remove the line or we can start updating it:

References:
https://github.com/pinterest/teletraan/pull/831/files?file-filters%5B%5D=.sql
https://github.com/pinterest/teletraan/pull/835/files#r631319018
https://github.com/pinterest/teletraan/blob/master/deploy-service/common/src/main/resources/sql/deploy.sql#L312